operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -2066,6 +2066,10 @@ metaMonitoring:
   serviceMonitor:
     # -- If enabled, ServiceMonitor resources for Prometheus Operator are created
     enabled: false
+    # -- Create standard Mimir alerts in Prometheus Operator via a PrometheusRule CRD
+    mimirAlerts: true
+    # -- Create standard Mimir recording rules in Prometheus Operator via a PrometheusRule CRD
+    mimirRules: true
```

Comment:
Should this be under `serviceMonitor`? It feels like it fits better under `metaMonitoring.prometheusRule`

Reply:
I'm also not sure whether want the alerts turned on by default. I'd say no because they are alerts and I'd rather not surprise users with random firing alerts.

Reply:
We can default to false for alerts, I'm comfortable with that.

Krajo and I discused putting it under under prometheusRule but went with ServiceMonitor due to 
```
# -- If enabled, ServiceMonitor resources for Prometheus Operator are created
    enabled: false
```
comment on ServiceMonitor. So since that's the piece that actually needs to be enabled to see/use these alerts and rules and the prometheusRule category starts to muddy the waters of "what's required" to just get a decent set of rules and alerts

Reply:
but we aren't creating service monitors, we are creating prometheus rules. it's weird enabling prometheus rules via the serviceMonitor section. The user can be collecting the metrics with their own service monitors.

> So since that's the piece that actually needs to be enabled to see/use these alerts and rules

I think we need both ServiceMonitor and PrometheusRule to see the rules/alerts and I don't think one is superior or a bigger prerequisite than the other.

My point is that that having PrometheusRules configured from two different sections in the yaml file is confusing (`serviceMonitor` and `prometheusRule`).

Reply:
furthermore, users will need to still toggle two booleans (serviceMonitor.enabled and serviceMonitor.mimirAlerts) to get a decent set of alerts. Is your point that having the two booleans under the same section makes it easier to understand?

Reply:
Without serviceMonitor enabled AND the recording rules, you don't have the metrics that the alerts use.

So my reasoning was: have a single toggle for enabling all of them, hence serviceMonitor.enabled . But also add the ability to disable parts of it - as you say , alerts might be annoying.

I guess an easy way out of this is to move them under PrometheusRule and add a validation that fails if you try to enable the rules and alerts without enabling serviceMonitor.

Reply:
Okay to clarify:  no validation surrounding the serviceMonitor being enabled is necessary, but it is required for the alerts, yes? so are we putting in the documentation that serviceMonitor.enabled: true is needed or am I wrong about that being needed?